src/main/java/com/cactus/systems/GroceryListSystem.java

```diff
@@ -152,14 +68,32 @@ public boolean newItem(String name){
      *
      * @return groceryItemNames
      * */
-    public ArrayList<String> getGroceryItemNames(){
+    public ArrayList<String> getGroceryItemNames(long userid){
```

Comment:
Is the UI meant to keep track of the user ID?

Reply:
It will get the userid from the UserSystem

Reply:
Does it make more sense for the controller to take care of that behind the scenes somehow? I'm not sure if the UI is supposed to handle anything other than literally just display and input.

Reply:
I guess I could have a currentLists hash stored? with the key as names and values as ids?